### DISCUSSION TITLE
Maya connector (native or legacy) won't install

### DISCUSSION THREAD

**Discussion by HereAtBRUHLabs** [Autodesk/maya-usd#3200] [Q&A]
I have been trying to install the Omniverse connector for Maya, all other software connectors inst5all fine but Maya gives me an error that says: Autodesk Maya is not currently installed. However, it is installed and updated. How fix?

**Comment by santosd** [comment]
Hi @HereAtBRUHLabs , you may need to reach out to Nvidia support as Autodesk doesn't develop the connector. I did a bit of searching on there forums and found this:
https://forums.developer.nvidia.com/t/maya-2024-mayaomniverse-connector/251317
If you are using Maya 2024, that could be the problem. Also might be good to ask your question on their forums.

Cheers!
